CHANGELOG.md

```diff
@@ -5,4 +5,6 @@ All notable changes to the Zowe Client Python SDK will be documented in this fil
 ## Recent Changes
 
 - Feature: Added a CredentialManager class to securely retrieve values from credentials and manage multiple credential entries on Windows [#134](https://github.com/zowe/zowe-client-python-sdk/issues/134)
-- Feature: Added method to load profile properties from environment variables
\ No newline at end of file
+- Feature: Added method to load profile properties from environment variables
+- Feature: Added method to Save profile properties to zowe.config.json file [#73](https://github.com/zowe/zowe-client-python-sdk/issues/73)
+- Feature : Added method to Save secure profile properties to vault [#72](https://github.com/zowe/zowe-client-python-sdk/issues/72)
```

Comment:
looks like there's an extra space on line 10 🙂

Reply:
Yes, the text is on the same line, but displayed on a new line on GitHub.